polyfill/test/helpers/temporalHelpers.js

```diff
@@ -117,6 +117,30 @@ var TemporalHelpers = {
     assert.sameValue(yearMonth.monthCode, monthCode, `${description} monthCode result`);
   },
 
+  /*
+   * assertZonedDateTime(zdt, year, ..., nanosecond[, description[, era, eraYear]]): XXX
```

Comment:
I hadn't needed this yet in other tests because I preferred asserting the value of `zdt.epochNanoseconds`, as it corresponds to the actual data model of ZonedDateTime. You might be able to do that in the toZonedDateTime() test as well?